Yikes, I wasn't aware we ran a python linter as part of the validation tests. @kastiglione mind taking a look at the issue?

```
18:02:35 ./utils/lldb/lldbToolBox.py:12:1: I100 Import statements are in the wrong order. import os should be before import lldb
18:02:35 ./utils/lldb/lldbToolBox.py:94:21: E261 at least two spaces before inline comment
```

--- a/utils/lldb/lldbToolBox.py
+++ b/utils/lldb/lldbToolBox.py
@@ -8,12 +8,13 @@ to the swift checkout.
 """
 
 import argparse
-import lldb
 import os
 import shlex
 import subprocess
 import sys
 import tempfile
+
+import lldb
 
 REPO_BASE = os.path.abspath(os.path.join(__file__, os.pardir, os.pardir,
                                          os.pardir, os.pardir))
@@ -91,7 +92,7 @@ def sequence(debugger, command, exec_ctx, result, internal_dict):
     for subcommand in command.split(';'):
         subcommand = subcommand.strip()
         if not subcommand:
-            continue # skip empty commands
+            continue  # skip empty commands
 
         ret = lldb.SBCommandReturnObject()
         interpreter.HandleCommand(subcommand, exec_ctx, ret)

